src/cirrus/quality_control.py

```diff
@@ -118,27 +140,32 @@ def main():
     opts = build_parser(sys.argv)
     if opts.only_changes:
         files = get_diff_files(None)
-        #we only want python modules
+        # we only want python modules
         for item in files:
             if not item.endswith('.py'):
                 files.remove(item)
         if not files:
             LOGGER.info("No modules have been changed.")
-            exit(0)
+            sys.exit(0)
     else:
         files = opts.files
-    #run all if none specified
+
+    pass_qc = True
+    # run all if none specified
     if not opts.pylint and not opts.pep8 and not opts.pyflakes:
-        run_pep8(opts.verbose, files=files)
-        run_pyflakes(opts.verbose, files=files)
-        run_pylint(files=files)
+        pass_qc = run_pep8(opts.verbose, files=files) and pass_qc
```

Comment:
I think its cleaner to do:
pass_pylint = blah
pass_pyflakes = blah 
pass_pep8 = blah 

if not all(pass_pylint, pass_pyflakes, pass_pep8):
     exit(90000000) 

It also gives a natural way of providing a verbose report to explain what actually failed